mmv1/third_party/terraform/tests/resource_compute_security_policy_test.go.erb

```diff
@@ -230,6 +230,30 @@ func TestAccComputeSecurityPolicy_withAdaptiveProtection(t *testing.T) {
 	})
 }
 
+<% unless version == 'ga' -%>
+func TestAccComputeSecurityPolicy_withAdaptiveProtectionAutoDeployConfig(t *testing.T) {
```

Comment:
If the new fields can be updated in-place, could you add a test that sets them to different values after the initial creation?

Reply:
It's possible to do this update test, but we don't find other references values to put there, we preferred to create a test with default values instead put random numbers on values.

Reply:
Hello there! I put an update step on the autoDeploy test, the update step test works on local environment, but the remove step I got an error, I don't know if it is possible to set the remove step on test:

![image](https://user-images.githubusercontent.com/120479960/213287973-a2f2f817-4368-4374-beac-0fe9c4efdedb.png)

Reply:
Ah yes, I think you're fine without a "clear" step. Nested fields like this can get weird because Terraform assumes `{}` to mean "unset" as opposed "set to empty". Your update test looks sufficient to me.